test/get.test.js

```diff
@@ -362,7 +362,7 @@ fastify.listen(0, err => {
     }, (err, response, body) => {
       t.error(err)
       t.strictEqual(response.statusCode, 200)
-      t.strictEqual(response.headers['content-length'], '0')
+      t.strictEqual(response.headers['content-length'], undefined)
```

Comment:
IMHO this is wrong.
The content length should always be set

Reply:
The spec allows for it to *not* be set -- https://tools.ietf.org/html/rfc2616#section-14.13

Reply:
yeah and in the rfc:

> Applications SHOULD use this field to indicate the transfer-length of the message-body

where "SHOULD" means:

> This word, or the adjective "RECOMMENDED", mean that there may exist valid reasons in particular circumstances to ignore a particular item, but the full implications must be understood and carefully weighed before choosing a different course

(https://tools.ietf.org/html/rfc2119)

Reply:
Yes, but it isn't a "MUST".